python/skupper_router/management/skrouter.json

```diff
@@ -1254,6 +1254,13 @@
                     "type": ["up", "down"],
                     "description": "The operational status of TCP socket listener: up - the service is active and incoming connections are permitted; down - the service is not active and incoming connection attempts will be refused.",
                     "create": false
+                },
+                "idleTimeoutSeconds": {
```

Comment:
We used to have an attribute called dataConnectionCount which used to be in the connector entity and @grs requested that it be moved to the router entity which we did. @grs, I just wanted to get your thoughts on whether this idleTimeoutSeconds should be in the router entity as well ?

Reply:
From a code standpoint it's easier to have it in the listener.  Having it in the listener also allows us to disable it per-service type.  More flexibility. You'll also notice that the AMQP listener and connector management object have the same 'idleTimeoutSeconds' attribute, so adding it to the tcpListener is consistent with that.

I don't see a compelling advantage to having a single global value for all services. Is there a use-case where it makes sense?

Reply:
The counter to that is whether we really envisage users wanting to set different values for this? What will the default be? Could there be a way to configure the default (i.e. a way to change it for all listeners in one place instead of changing the configuration of each individual listener)?

What advice would we give users about when to use this option and what to set it to? And can we make it so that in our opinion the majority of users can ignore it?